Modules/Video/BridgeOpenCV/test/itkOpenCVVideoIOTest.cxx

```diff
@@ -38,7 +38,7 @@ ImageType::Pointer
 itkImageFromBuffer(itk::OpenCVVideoIO::Pointer opencvIO, void * buffer, size_t bufferSize)
 {
   // Set up for incoming image
-  constexpr ImageType::SizeType  size{ opencvIO->GetDimensions(0), opencvIO->GetDimensions(1) };
+  const ImageType::SizeType      size{ opencvIO->GetDimensions(0), opencvIO->GetDimensions(1) };
```

Comment:
Sorry!

Reply:
No problem, of course. For unrelated reason, I had OpenCV bridge enabled locally, so I noticed this quickly. Do you generally make this refactoring changes with as many non-default modules enabled? Or just the default ones?